seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/checkpoint/CheckpointCoordinator.java

```diff
@@ -767,14 +767,16 @@ public synchronized void completePendingCheckpoint(CompletedCheckpoint completed
         final long checkpointId = completedCheckpoint.getCheckpointId();
         completedCheckpointIds.addLast(String.valueOf(completedCheckpoint.getCheckpointId()));
         try {
-            byte[] states = serializer.serialize(completedCheckpoint);
-            checkpointStorage.storeCheckPoint(
-                    PipelineState.builder()
-                            .checkpointId(checkpointId)
-                            .jobId(String.valueOf(jobId))
-                            .pipelineId(pipelineId)
-                            .states(states)
-                            .build());
+            if (completedCheckpoint.getCheckpointType().notCompletedCheckpoint()) {
```

Comment:
Can you add an annotation to this method indicating that it represents a batch type

https://github.com/apache/seatunnel/blob/058f5594a3691f3f187c63a5d3bcb1a0e29b6a53/seatunnel-engine/seatunnel-engine-core/src/main/java/org/apache/seatunnel/engine/core/checkpoint/CheckpointType.java#L94

Reply:
Because the judgment here is for batch processing, but currently the checkpoint type cannot be associated with batch processing